api/views.py

```
@@ -0,0 +1,115 @@
+)
+
+SYSTEM_USER_ID = UUID(getattr(settings, 'SYSTEM_USER_ID'))
+
+
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+
+class PatientViewSet(viewsets.ModelViewSet):
+    queryset = Patient.objects.all().order_by('-created_at')
+    serializer_class = PatientSerializer
+    permission_classes = [IsAuthenticated]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)